javascript/node-fetch/security/audit/node-fetch-cookie-leak.yaml

```diff
@@ -0,0 +1,42 @@
+rules:
+  - id: node-fetch-cookie-leak
+    patterns:
+      - pattern-either:
+        - pattern: |
+            fetch(..., { ..., headers: {..., 'Cookie': ..., ...}, ... })
+        - pattern: |
+            const $HEADERS = {..., 'Cookie': ..., ...}
+            ...
+            fetch(..., { headers: $HEADERS }, ...)
+        - pattern: |
+            const $OPTIONS = {..., headers: $HEADERS, ...}
```

Comment:
this case doesn't guarantee cookies are sent, does it?  While $HEADERS might be set in one of the other `pattern-either` definitions, this pattern can match `options` with no `Cookie` key.  We should add a test case for same.

Reply:
Fixed! I'll send you the updated rule and test on slack